src/arcade/global.json

```diff
@@ -1,14 +1,14 @@
 {
   "sdk": {
-    "version": "10.0.100-preview.5.25265.106",
+    "version": "10.0.100-preview.5.25302.104",
     "rollForward": "latestFeature"
   },
   "tools": {
-    "dotnet": "10.0.100-preview.5.25265.106"
+    "dotnet": "10.0.100-preview.5.25302.104"
```

Comment:
this one is wrong, it should've been preview.6 not preview.5: https://github.com/dotnet/dotnet/pull/951